docs/DesignDocs/Protected_Code_Loader_Proposal.md

```diff
@@ -0,0 +1,161 @@
+Open Enclave Protected Code Loader
+====
+
+Enabling Enclave Code Confidentiality in Open Enclave.
+
+# Motivation
```

Comment:
Is there a specific object to provide some kind of developer experience parity with the Intel SDK PCL feature? It would be useful to call them out as we consider the design. 

For example, is there a desire to preserve the mechanism for delivering PCL keys to the target device via key sealing (i.e. A dev can reuse a previously developed key sealing enclave and it would still work with the updated OE SDK based PCL enclave. That would also create a specific requirement on the Sealing API feature also under discussion.